lib/core/plugins/context.js

```diff
@@ -120,17 +120,16 @@ class PluginContext {
       };
 
       /* context.log ======================================================== */
-
       Reflect.defineProperty(this, 'log', {
         enumerable: true,
         get: () => {
           return {
-            debug: (...args) => kuzzle.log.debug(...args),
-            error: (...args) => kuzzle.log.error(...args),
-            info: (...args) => kuzzle.log.info(...args),
-            silly: (...args) => kuzzle.log.silly(...args),
-            verbose: (...args) => kuzzle.log.verbose(...args),
-            warn: (...args) => kuzzle.log.warn(...args)
+            debug: msg => kuzzle.log.debug(`[plugin-${pluginName}] ${msg}`),
```

Comment:
This implementation break the logger public API since this is not a variadic funtion anymore. Maybe something like this could do the trick:
```suggestion
            debug: (...args) => kuzzle.log.debug(`[plugin-${pluginName}] `, args),
```

Reply:
In the current documentation, the log methods only takes one argument so it's not a breaking change.
See @scottinet comments here: https://github.com/kuzzleio/kuzzle/pull/1571#discussion_r378295795